apps/admin/config.forge.js

```diff
@@ -0,0 +1,44 @@
+module.exports = {
+  packagerConfig: {},
+  makers: [
+    {
+      name: "@electron-forge/maker-squirrel",
```

Comment:
What is this file for?

Reply:
Yeah, `Electron-Forge` is the package that runs our app.  It's kinda like `create-react-app`, just for electron.